ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/AbstractEngineNewPayload.java

```diff
@@ -178,6 +179,19 @@ public JsonRpcResponse syncResponse(final JsonRpcRequestContext requestContext)
               .map(Bytes::fromHexString)
               .map(in -> TransactionDecoder.decodeOpaqueBytes(in, EncodingContext.BLOCK_BODY))
               .collect(Collectors.toList());
+      transactions.forEach(
+          transaction ->
+              mergeCoordinator
+                  .getEthScheduler()
+                  .scheduleTxWorkerTask(
+                      () -> {
+                        Address sender = transaction.getSender();
+                        LOG.atTrace()
+                            .setMessage("The sender for transaction {} is calculated : {}")
+                            .addArgument(transaction.getHash())
```

Comment:
```suggestion
                            .addArgument(transaction::getHash)
```

Reply:
I prefer the initial one to be honest, I think we should keep it simple, transaction::getHash will be transformed into transaction.getHash so why not just use transaction.getHash.

Reply:
there is a semantic difference when using `LOG.atTrace` with lambda or direct method call:

- `.addArgument(transaction.getHash())` means that the methos is always executed even if the log level is > TRACE
- .addArgument(transaction::getHash) means that the method is only executed if the level is <= TRACE

and since `transaction.getHash()` is potentially expensive method, it is worth to avoid calling it when the log is not done.